tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -2178,14 +2176,15 @@ declare namespace ts {
         declaration?: SignatureDeclaration;
     }
     enum InferencePriority {
-        NakedTypeVariable = 1,
-        MappedType = 2,
-        ReturnType = 4,
+        Contravariant = 1,
```

Comment:
@ahejlsberg this changed in #21157. Are these changes intended to be part of the public API?